.github/jobs/uwp.yml

```
@@ -0,0 +1,26 @@
+parameters:
+- name: name
+  type: string
+- name: platform
+  type: string
+- name: jsEngine
+  type: string
+
+jobs:
+- job: ${{parameters.name}}
+  timeoutInMinutes: 15
+
+  pool:
+    vmImage: windows-latest
+
+  steps:
+  - script: cmake -B Build/uwp -A ${{parameters.platform}} -D NAPI_JAVASCRIPT_ENGINE=${{parameters.jsEngine}} -D CMAKE_SYSTEM_NAME=WindowsStore -D CMAKE_SYSTEM_VERSION=10.0 
+    displayName: 'Configure CMake'
+
+  - task: MSBuild@1
+    inputs:
+      solution: 'Build/uwp/JsRuntimeHost.sln'
+      maximumCpuCount: true
+      configuration: 'Debug'
+    displayName: 'Build Solution'
```

nit: just to be consistent
```suggestion
  steps:
  - script: cmake -B Build/UWP -A ${{parameters.platform}} -D NAPI_JAVASCRIPT_ENGINE=${{parameters.jsEngine}} -D CMAKE_SYSTEM_NAME=WindowsStore -D CMAKE_SYSTEM_VERSION=10.0 
    displayName: 'Configure CMake'

  - task: MSBuild@1
    inputs:
      solution: 'Build/UWP/JsRuntimeHost.sln'
      maximumCpuCount: true
      configuration: 'Debug'
    displayName: 'Build Solution'
```